export/index.html

```diff
@@ -240,6 +252,76 @@ <h2>Message log</h2>
         return new Uint8Array([...s, ...r]);
       }
 
+      /**
+       * Function to verify enclave signature on import bundle received from the server. 
+       */
+       const verifyEnclaveSignature = async (enclaveQuorumPublic, publicSignature, publicKey) => {
```

Comment:
do you know if it's possible to use [JS modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) within an iFrame? I'm seeing a lot of duplicate code which will become harder to maintain. It'd be awesome if we could use a module system although it might not be possible,

Reply:
yep! the HpkeEncrypt and HpkeDecrypt code are actually already JS modules. at some point early on, i tried to DRY up the shared code across the 3 pages but it is a lot of refactoring. as some follow ups, i think we modularize:
- part 1: import/index.html and import/standalone.html
- part 2: import and export
- part 3: import/export and auth